test.sh

```diff
@@ -64,10 +97,107 @@ function run_test_coverage() {
   goveralls -v -coverprofile=gover.coverprofile -service=travis-pro
 }
 
+#
+# Main CLI Parser
+#
+USAGE="$(cat -- <<-EOM
+
+Usage:
+Boulder test suite CLI, intended to be run inside of a Docker container:
+
+  docker-compose run --use-aliases boulder ./$(basename "${0}") [OPTION]...
+
+With no options passed: runs standard battery of tests (lint, unit, and integation)
+    -l, --lints                           Adds lint to the list of tests to run
+    -u, --unit                            Adds unit to the list of tests to run
+    -d, --unit-test-filter <DIRECTORY>    Run unit tests for a specific directory
+    -e, --enable-unit-race                Enables -race flag for all unit tests
```

Comment:
Let's have a single `--race` flag that applies to both unittests and integration tests. Right now integration-test.py checks the `RACE` env var; we could either set that in `test.sh`, or change that to be a flag, and add the flag in `test.sh`.